src/components/category.js

```diff
@@ -220,9 +218,13 @@ Category.propTypes = {
   perLine: PropTypes.number.isRequired,
   emojiProps: PropTypes.object.isRequired,
   recent: PropTypes.arrayOf(PropTypes.string),
+  notFound: PropTypes.shape({
+    notFoundImg: PropTypes.string.isRequired,
+  }),
 }
 
 Category.defaultProps = {
   emojis: [],
   hasStickyPosition: true,
+  notFound: { notFoundImg: '' },
```

Comment:
I think we should be able to remove this? `notFound` should not require a default value, since it's a purely optional prop.